src/backend/joanie/core/api/client/__init__.py

```diff
@@ -1533,11 +1533,7 @@ class NestedOrderCourseViewSet(NestedGenericViewSet, mixins.ListModelMixin):
     ordering = ["-created_on"]
     queryset = (
         models.Order.objects.filter(
```

Comment:
In commit body (5a5d83aadfe119150e2e02c20391eba25e3d73e4):

`s/As we now allows/As we now allow`